lib/exercism/active_record_citext.rb

```diff
@@ -1,4 +0,0 @@
-# from http://gray.fm/2013/09/17/unknown-oid-with-rails-and-postgresql/
-ActiveRecord::ConnectionAdapters::PostgreSQLAdapter.tap do |klass|
-  klass::OID.register_type('citext', klass::OID::Identity.new)
-end
```

Comment:
@kytrinyx citext is now supported in Rails 4.2 Please check https://github.com/rails/rails/pull/12523. So I guess we are fine after removing this file.